src/Application/Common/Results/ErrorCode.cs

```diff
@@ -33,6 +33,7 @@ public enum ErrorCode
     ItemBroken,
     ItemDisabled,
     ItemNotBuyable,
+    ItemNotSellable,
```

Comment:
Try to keep the enum alphabetically ordered. Same for their associated methods.

Reply:
done